install-and-configure/advanced-configuration/resource-consumption.md

```diff
@@ -4,63 +4,61 @@ Kubecost can run on clusters with thousands of nodes when resource consumption i
-* The default value is: `300s`
-* This can be tuned with the Helm value:
-  * `--set kubecostModel.etlResolutionSeconds=600`
+Lowering query resolution will reduce memory consumption but will cause short running pods to be sampled and rounded to the nearest interval for their runtime. The default value is: `300s`. This can be tuned with the Helm value:
 
-## Longer scrape interval
+* `--set kubecostModel.etlResolutionSeconds=600`
 
-* Fewer data points scraped from Prometheus means less data to collect and store, at the cost of Kubecost making estimations that possibly miss spikes of usage or short running pods
-* The default value is: `60s`
-* This can be tuned in our [Helm values](https://github.com/kubecost/cost-analyzer-helm-chart/blob/v1.93.2/cost-analyzer/values.yaml#L389) for the Prometheus scrape job.
+## Lengthen scrape interval
+
+Fewer data points scraped from Prometheus means less data to collect and store, at the cost of Kubecost making estimations that possibly miss spikes of usage or short running pods. The default value is: `60s`. This can be tuned in our [Helm values](https://github.com/kubecost/cost-analyzer-helm-chart/blob/v1.93.2/cost-analyzer/values.yaml#L389) for the Prometheus scrape job.
 
 ## Disable or stop scraping node exporter
 
-* Node-exporter is optional. Some health alerts will be disabled if node-exporter is disabled, but savings recommendations and core cost allocation will function as normal
-* This can be disabled with the following [Helm values](https://github.com/kubecost/cost-analyzer-helm-chart/blob/v1.93.2/cost-analyzer/values.yaml#L442):
-  * `--set prometheus.server.nodeExporter.enabled=false`
-  * `--set prometheus.serviceAccounts.nodeExporter.create=false`
+Node-exporter is optional. Some health alerts will be disabled if node-exporter is disabled, but savings recommendations and core cost allocation will function normally. This can be disabled with the following [Helm values](https://github.com/kubecost/cost-analyzer-helm-chart/blob/v1.93.2/cost-analyzer/values.yaml#L442):
+
+* `--set prometheus.server.nodeExporter.enabled=false`
+* `--set prometheus.serviceAccounts.nodeExporter.create=false`
 
 ## Enable Filestore
 
-* Filestore is an improvement over our legacy in-memory ETL store of Prometheus data. It was optional in versions up to v1.94.0, but will become the default afterward.
-* This can be enabled on older versions with the [Helm value](https://github.com/kubecost/cost-analyzer-helm-chart/blob/v1.94.3/cost-analyzer/values.yaml#L271):
-  * `--set kubecostModel.etlFileStoreEnabled=true`
+Filestore is an improvement over our legacy in-memory ETL store of Prometheus data. It was optional in versions up to v1.94.0, but is now the default. This can be enabled on older versions with the [Helm value](https://github.com/kubecost/cost-analyzer-helm-chart/blob/v1.94.3/cost-analyzer/values.yaml#L271):
+
+* `--set kubecostModel.etlFileStoreEnabled=true`
```

Comment:
I recommend deleting this section! This has been the default value since at least v1.96.